src/tests/omni_lock.rs

```diff
@@ -199,6 +301,95 @@ fn test_omnilock_transfer_from_multisig() {
     ctx.verify(tx, FEE_RATE).unwrap();
 }
 
+#[test]
+fn test_omnilock_transfer_from_multisig_wl() {
+    let lock_args = vec![
+        ACCOUNT0_ARG.clone(),
+        ACCOUNT1_ARG.clone(),
+        ACCOUNT2_ARG.clone(),
+    ];
+    let multi_cfg = MultisigConfig::new_with(lock_args, 0, 2).unwrap();
+    let mut cfg = OmniLockConfig::new_multisig(multi_cfg);
```

Comment:
The `auth_content` should be `H160::zero()` so that the lock script is unlocked by "admin mode".